docusaurus.config.js

```diff
@@ -224,10 +223,36 @@ const config = {
             from: "/blog/posts/2023-07-13-installer-security-fixes/",
             to: "/blog/2023/07/13/installer-security-fixes/",
           },
+          /* user redirects */
           {
             from: "/docs/user/announcements.html",
             to: "/announcements/",
           },
+          /* organization redirects */
+          {
+            from: "/docs/orga/governance.html",
+            to: "/community/governance/",
+          },
+          {
+            from: "/docs/orga/subteams.html",
+            to: "/community/subteams/",
+          },
+          {
+            from: "/docs/orga/joining-the-team.html",
+            to: "/community/join/",
+          },
+          {
+            from: "/docs/orga/cfep-index.html",
+            to: "/community/cfep/",
+          },
+          {
+            from: "/docs/orga/getting-in-touch.html",
+            to: "/community/get-in-touch/",
+          },
+          {
+            from: "/docs/contracting/00_intro.html",
```

Comment:
```suggestion
            from: ["/docs/contracting/index.html", "/docs/contracting/00_intro.html"],
```

I think? Maybe add both just in case.

Reply:
Fixed